src/pages/_app.js

```diff
@@ -145,33 +163,14 @@ function MyApp({ Component, pageProps, intercom, admin, irods, tools }) {
     );
 }
 
-MyApp.getInitialProps = async (ctx) => {
-    const { publicRuntimeConfig = {} } = getConfig() || {};
```

Comment:
Was there a reason to move this stuff into the `useEffect`?  Personally, I liked here, since it was a little more out of the way.  Minor thing though.

Reply:
Yes. I got an error mentioned here - https://github.com/isaachinman/next-i18next/issues/124. I could make it work with this change.